src/mmg.jl

```diff
@@ -336,23 +467,46 @@ Maneuvering parameters of target ship for MMG 3DOF simulation.
 end
 
 """
-    mmg_3dof_simulate(time_list, n_p_list, δ_list, basic_params, maneuvering_params, [, u0, v0, r0, x0, y0, Ψ0, ρ, algorithm, reltol, abstol]) -> u, v, r, x, y, Ψ, δ, n_p
+Sturucture parameters of target ship for MMG 3DOF simulation.
+
+# Arguments
+- `A_OD::T`
+- `A_F::T`
+- `A_L::T`
+- `H_BR::T`
+- `H_C::T`
+- `C::T`
+"""
+@with_kw mutable struct Mmg3DofStructureParams{T}
+    A_OD::T
+    A_F::T
+    A_L::T
+    H_BR::T
+    H_C::T
+    C::T
+end
```

Comment:
Various parameters need to be divided into the parameters needed for the simulation and the information to define them.

From your PR, I think that the following struct should be defined as essential information for MMG simulation.

```julia
@with_kw mutable struct Mmg3DofWindForceMomentParams{T}
    A_F::T
    A_L::T
    C_X_wind::T
    C_Y_wind::T
    C_N_wind::T
end
```

Reply:
Thank you for the comments!
It seems difficult to change `Mmg3DofStructureParams`to`Mmg3DofWindForceMomentParams`, because `C_X_wind`,`C_Y_wind`,`C_N_wind` are calculated sequentially using `Mmg3DofStructureParams` in `mag_3dof_model!`.
We need to have a discussion about this.